examples/cifar10/cifar10-real-world/workspaces/secure_project.yml

```diff
@@ -0,0 +1,82 @@
+api_version: 3
+name: secure_project
+description: NVIDIA FLARE sample project yaml file for CIFAR-10 example
+
+participants:
+  - name: overseer.example.com
+    type: overseer
+    org: nvidia
+    protocol: https
+    api_root: /api/v1
+    port: 8443
+  - name: localhost
+    type: server
+    org: nvidia
+    fed_learn_port: 8102
+    admin_port: 8103
+  - name: site-1
+    type: client
+    org: nvidia
+  - name: site-2
+    type: client
+    org: nvidia
+  - name: site-3
+    type: client
+    org: nvidia
+  - name: site-4
+    type: client
+    org: nvidia
+  - name: site-5
+    type: client
+    org: nvidia
+  - name: site-6
+    type: client
+    org: nvidia
+  - name: site-7
+    type: client
+    org: nvidia
+  - name: site-8
+    type: client
+    org: nvidia
+  - name: admin@nvidia.com
+    type: admin
+    org: nvidia
+    role: project_admin
+
+# The same methods in all builders are called in their order defined in builders section
+builders:
+  - path: nvflare.lighter.impl.workspace.WorkspaceBuilder
+    args:
+      template_file: master_template.yml
+  - path: nvflare.lighter.impl.template.TemplateBuilder
+  - path: nvflare.lighter.impl.static_file.StaticFileBuilder
+    args:
+      config_folder: config
+      overseer_agent:
+        path: nvflare.ha.dummy_overseer_agent.DummyOverseerAgent
+        # if overseer_exists is true, args here are ignored.  Provisioning
+        #   tool will fill role, name and other local parameters automatically.
+        # if overseer_exists is false, args in this section will be used.
+        overseer_exists: false
+        args:
+           sp_end_point: localhost:8102:8103
+           heartbeat_interval: 6
+
+      snapshot_persistor:
+        path: nvflare.app_common.state_persistors.storage_state_persistor.StorageStatePersistor
+        args:
+          uri_root: /
+          storage:
+            path: nvflare.app_common.storages.filesystem_storage.FilesystemStorage
+            args:
+              root_dir: /tmp/nvflare/snapshot-storage
+              uri_root: /
```

Comment:
@IsaacYangSLA  are we reading this part?

Reply:
I mean does `StaticFileBuilder` take this `snapshot_persistor` arg?

Reply:
Seems that this argument is deprecated. It's usage is commented out https://github.com/NVIDIA/NVFlare/blob/a962b00e61861ea5e8f8174c42e23ed98038001c/nvflare/lighter/impl/static_file.py#L152.
Should I remove it?

Reply:
It's also used in this example here https://github.com/NVIDIA/NVFlare/blob/a962b00e61861ea5e8f8174c42e23ed98038001c/examples/federated-policies/project.yml#L55

Reply:
From @IsaacYangSLA : The snapshot_persistor is no longer used in project.yml.  Currently, the logic is to generate the default (from template) in local folder.

I'll remove it from my project.yml